lib/activesupport/all/activesupport.rbi

```diff
@@ -96,5 +96,215 @@ class NilClass
       b: T.nilable(T.proc.params(arg0: T.untyped).returns(T.untyped))
     ).returns(NilClass)
   end
-  def try!(method_name = nil, *args, &b) ; end
+  def try!(method_name = nil, *args, &b); end
+end
+
+class String
+  sig { returns(T::Boolean) }
+  def acts_like_string?; end
+
+  sig { params(position: T.any(Integer, String, Regexp)).returns(T.nilable(String)) }
```

Comment:
Why not support the `Range` case as well, for the position argument?

Reply:
Good catch, I missed that Ranges were accepted. Fixed 👍